src/test/kotlin/fi/metatavu/muisti/api/test/functional/VisitorSessionV2TestsIT.kt

```diff
@@ -0,0 +1,516 @@
+package fi.metatavu.muisti.api.test.functional
+
+import fi.metatavu.muisti.api.client.models.*
+import org.awaitility.Awaitility
+import org.junit.Assert.*
+import org.junit.Test
+import java.time.Duration
+import java.time.OffsetDateTime
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+/**
+ * Test class for testing visitor sessions API
```

Comment:
Versioning could be added to docs too for clarification?
```suggestion
 * Test class for testing visitor sessions API V2
```